### DISCUSSION TITLE
Github pages failing to deploy

### DISCUSSION THREAD

**Discussion by maysaleba** [community/community#44821] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Getting this error trying to update my website for the last hour:

![image](https://user-images.githubusercontent.com/80749020/213548417-d137a03e-5702-4179-b08a-932eaed5c6fb.png)

**Comment by Parking-Master** [comment]
Me too, retried 3 times and still the same thing. They’re fixing it now and you can see status records at [githubstatus.com](https://githubstatus.com). Should be fixed soon, don’t worry

**Comment by Parking-Master** [comment, marked as answer]
The issue has been fixed now.

**Comment by queenofcorgis** [comment]
Happy to hear this has been resolved, @maysaleba. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
